disnake/ext/commands/interaction_bot_base.py

```diff
@@ -231,32 +269,127 @@ def command_sync_flags(self) -> CommandSyncFlags:
         """
         return CommandSyncFlags._from_value(self._command_sync_flags.value)
 
-    def application_commands_iterator(self) -> Iterable[InvokableApplicationCommand]:
-        return chain(
-            self.all_slash_commands.values(),
-            self.all_user_commands.values(),
-            self.all_message_commands.values(),
-        )
+    @property
+    def all_app_commands(self) -> MappingProxyType[AppCmdIndex, InvokableApplicationCommand]:
+        """MappingProxyType[:class:`AppCmdIndex`, :class:`InvokableApplicationCommand`]:
+        A mapping proxy with all application commands the bot has.
```

Comment:
```suggestion
        """Mapping[:class:`AppCmdIndex`, :class:`InvokableApplicationCommand`]:
        A read-only mapping with all application commands the bot has.
```
To be consistent with other usages of `MappingProxyType`

Reply:
https://github.com/DisnakeDev/disnake/pull/1107/commits/a5802385de007644aacf5d5a03a3b2c685ea1b41